CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
-2.6.1 (July 7, 2023)
+2.7.0 (In Progress)
 ===================
 
+ECMAScript Module Support
+-------------------------
+
+Starting from this version, the SDK can now be imported as an [ECMAScript Module](https://nodejs.org/api/esm.html) (ESM) using the `import` keyword. The SDK can continue to be imported as a [CommonJS Module](https://nodejs.org/api/modules.html) using the `require` keyword.
```

Comment:
This might be a little confusing since customers are already able to use `import` with the latest version. Should we make this a bit simpler since we are already providing a link regarding esm? Something like
```
Starting from this version, the SDK can now be imported as an [ECMAScript Module](https://nodejs.org/api/esm.html) (ESM)
```